src/unix/linux_like/linux/mod.rs

```diff
@@ -19,6 +19,7 @@ pub type pthread_key_t = ::c_uint;
 pub type pthread_once_t = ::c_int;
 pub type pthread_spinlock_t = ::c_int;
 pub type __kernel_fsid_t = __c_anonymous__kernel_fsid_t;
+pub type __kernel_clockid_t = ::c_int;
```

Comment:
the `__kernel_fsid_t` symbol is not in the semver file, so I didn't add `__kernel_clockid_t` either. But maybe they should both be added?

Reply:
Would you be able to add both? Even though they're dunder types, it's nice to at least make it clear when they disappear.

Reply:
sure, I added them